src/workloads/sharding/ReshardCollectionWithIndexes.yml

```diff
@@ -0,0 +1,175 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  This test expects the target cluster is created using ebs snapshot with 1 billion 1KB records 
```

Comment:
Can you make it clearer that the cluster starts with all records in the same shard?

Reply:
Done

Reply:
Added